routes.js

```diff
@@ -5,6 +5,6 @@ const routes = module.exports = nextRoutes();
 // ========================= APP ROUTES =====================
 routes.add('home', '/', 'index');
 routes.add('panel', '/panel', 'panel');
-routes.add('indicator', '/panel/:indicator', 'panel');
+routes.add('indicator', '/panel/:indicator', 'indicator');
 routes.add('compare', '/compare', 'compare');
```

Comment:
Do we need this route `/compare`? As long as we are going to use the `/panel/:indicator` with the `:indicator` param as an array I don't think so.